### ISSUE TITLE
add feature resize columns and rows.

### ISSUE THREAD

**Issue by devinschumacher** [janisdd/vscode-edit-csv#111]
### Version?
- vs code plugin <--
- browser
- both?

### Feature

add feature resize columns and rows. my spreadsheets have long values. the columns are enormous. how is there now right click > resize??

**Comment by janisdd** [comment]
Not sure what you mean... like in excel, you can resize the columns by clicking and dragging between the column borders?

**Comment by janisdd** [comment]
Ah yes, have seen time sometimes... specifying a width via the context menu seems a good idea. However, I'm wondering how the ui should look like.

Actually, there is a feature that already fixes this... the setting `doubleClickColumnHandleForcedWith`. When you double click on the handle it shrinks the column to the configured size (default 200px) (if the current size is the auto size).
But I link a ui option would also be good. Maybe we can use the value from `doubleClickColumnHandleForcedWith` and don't need a value input?

(As I checked the `doubleClickColumnHandleForcedWith` I found that this is broken after you use the btn `Resize all column widths to match their content`.)
